src/imputation/expansion_imputation.py

```diff
@@ -58,6 +58,9 @@ def evaluate_imputed_ixx(
 def split_df_on_trim(df: pd.DataFrame, trim_bool_col: str) -> pd.DataFrame:
     """Splits the dataframe in based on if it was trimmed or not"""
 
+    # TODO: remove this temporary fix to cast Nans to False
+    df[trim_bool_col].fillna(False, inplace=True)
```

Comment:
@AnneONS does this need to be/can this be removed before we merge?

Reply:
no, I'm not sure how we would immediately, so shall we leave it in?

Reply:
Yeh that's fine - so long as it doesn't have any impact downstream